dashboards/assisted-service/grafana-dashboard-assisted-rds-postgres.configmap.yaml

```diff
@@ -1040,7 +1061,7 @@ data:
             "options": [],
             "query": "prometheus",
             "refresh": 1,
-            "regex": "/^app-sre-prod-01-prometheus$/",
```

Comment:
this is the relevant change

Reply:
In app-interface, Im seeing some `appsrep09ue1` as well. Not sure if they are relevant and if we should include them with a `d` in the regexp